src/main/java/com/vonage/quickstart/insight/AdvancedInsightAsync.java

```diff
@@ -42,7 +42,7 @@ public static void main(String... args) {
 
         AdvancedInsightRequest request = AdvancedInsightRequest.builder(INSIGHT_NUMBER)
                 .async(true)
-                .callback(ASYNC_CALLBACK_URL)
+                .callback("https://demo.ngrok.io/webhooks/insight")
```

Comment:
We should be constructing the URL for the route, not hard-coding it.

Reply:
Fixed per the new spec.